apps/back/config/bundles.php

```diff
@@ -7,4 +7,5 @@
     Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle::class => ['dev' => true, 'test' => true],
     Symfony\Bundle\TwigBundle\TwigBundle::class => ['all' => true],
     Symfony\Bundle\SecurityBundle\SecurityBundle::class => ['all' => true],
+    Symfony\Bundle\WebProfilerBundle\WebProfilerBundle::class => ['dev' => true, 'test' => true],
```

Comment:
```suggestion
    Symfony\Bundle\WebProfilerBundle\WebProfilerBundle::class => ['dev' => true],
```
I think you does not needs profiling in test environment 😉